sites/mosul/configs/openmrs/initializer_config/liquibase/liquibase.xml

```diff
@@ -89,13 +109,22 @@
             <!-- Only running this to remove previously recorded obs -->
             DELETE FROM obs WHERE value_coded = (SELECT concept_id FROM concept WHERE uuid = '80945AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA');
 
+            DELETE FROM drug WHERE concept_id = (SELECT concept_id FROM concept WHERE uuid = '80945AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA');
+
             <!-- Delete concept -->
             DELETE FROM concept WHERE uuid = '80945AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
         </sql>
     </changeSet>
 
     <!-- DELETE Severe Pain CIEL concept -->
     <changeSet id="delete_ciel_concept_severe_pain" author="pirupius">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                SELECT CASE WHEN COUNT(*) > 0 THEN 1 ELSE 0 END
+                FROM concept
+                WHERE uuid = '163477AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
+            </sqlCheck>
+        </preConditions>
```

Comment:
Similar to the Nystatin drug, this precondition exists for the Severe pain concept.